_data/nav.yml

```diff
@@ -830,6 +835,16 @@
 
 - url: "release-notes"
   pages:
+  - url: "release-notes56"
+    pages:
+    - url: "#Overview"
+    - url: "#New features"
+    - url: "#Enhancements"
+    - url: "#Accompanying Premium Plugin changes"
+    - url: "#General bug fixes"
```

Comment:
I think this is missing a link to `#securityfixes`

Reply:
```suggestion
    - url: "#General bug fixes"
    - url: "#Security fixes"
```

Reply:
Thanks @JamesToohey